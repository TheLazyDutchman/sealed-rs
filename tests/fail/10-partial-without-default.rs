use sealed::sealed;

#[sealed(partial)]
pub trait A {
	#[seal]
	fn sealed();
}

fn main() {}
