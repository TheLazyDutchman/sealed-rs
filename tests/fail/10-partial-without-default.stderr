error: This function is sealed from implementation, but it does not have a default implementation. This effectively seals the entire trait, which would be clearer to do by not having the trait seal be partial.
 --> tests/fail/10-partial-without-default.rs:6:5
  |
6 |     fn sealed();
  |        ^^^^^^
